build.gradle

```diff
@@ -136,6 +144,7 @@ publishing {
         node.appendNode('name', 'Elastic JNA Distribution')
         node.appendNode('description',
           'A build of jna which supports all platforms supported by Elasticsearch')
+        node.appendNode('url', 'https://github.com/elastic/jna-build')
```

Comment:
It kept failing without it when I tried to close the repository.  I can regenerate and take a screenshot.

I thought ours would be more appropriate since we actually packaged it, I'm fine with the upstream. Will change.